src/net40/Radical.Windows.Presentation/Services/ConventionsHanlder.cs

```diff
@@ -78,11 +78,21 @@ public ConventionsHanlder(IMessageBroker broker, IReleaseComponents releaser, Bo
                         }
 
                         this.releaser.Release(vm);
+                        var disposableViewModel = vm as IDisposable;
+                        if( disposableViewModel != null)
+                        {
+                            disposableViewModel.Dispose();
```

Comment:
what about using the null propagation `disposableViewModel?.Dispose();`?

apart of if you like or not the contractions, can we start develop c#6 code and force to use >=vs2015?